File: day-4-java-adv/Assignment5_OnlineShopping/Product.java
// Product class for Assignment 5
class Product {
    int id;
    String name;
    double price;

    Product(int id, String name, double price) {
        this.id = id;
        this.name = name;
        this.price = price;
    }

    void displayProduct() {
        System.out.println("Product ID: " + id);
        System.out.println("Name: " + name);
        System.out.println("Price: " + price);
    }
}

File: Day-26,27 react/src-26/components/Product.js
function Product(props) {
  return (
    <div>
      <h2>Product Details</h2>

      <p>Product : {props.name}</p>
      <p>Price : {props.price}</p>
      <p>Brand : {props.brand}</p>
      <p>Rating : {props.rating}</p>
      <p>Stock : {props.stock}</p>
    </div>
  );
}

export default Product;